network-store-iidm-impl/pom.xml

```diff
@@ -116,6 +116,11 @@
             <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why do you need this ? To remove ?

Reply:
Here https://github.com/powsybl/powsybl-core/pull/3020, it is said it's required. But is it really ?

Reply:
it is for some test from powsybl that will need it but I removed them because it was not working
I remove